src/azul/__init__.py

```diff
@@ -190,12 +190,12 @@ def num_dss_workers(self) -> int:
 
     @property
     def external_lambda_role_assumptors(self) -> MutableMapping[str, List[str]]:
-        try:
-            accounts = os.environ['AZUL_EXTERNAL_LAMBDA_ROLE_ASSUMPTORS']
-        except KeyError:
-            return {}
-        else:
+        accounts = os.environ.get('AZUL_EXTERNAL_LAMBDA_ROLE_ASSUMPTORS')
+        # This env variable should only be set for HCA deployments
+        if accounts is not None and config.domain_name.endswith('explore.data.humancellatlas.org'):
```

Comment:
Is there a way to avoid hard-coding the URL here? What if you just left this file the way it was except for the comment on line 194. The HCA deployments will still have this environment variable set AFAICT, and everything should just work. What do you think of that approach?

Otherwise, this looks good.